packages/xrpl/src/Wallet/fundWallet.ts

```diff
@@ -19,6 +19,17 @@ const INTERVAL_SECONDS = 1
 // Maximum attempts to retrieve a balance
 const MAX_ATTEMPTS = 20
 
+export const DEFAULT_MEMO = {
+  data: 'xrpl-js',
+  type: 'user-agent',
```

Comment:
I think it would be good to write/start an [XLS-Standard](https://github.com/XRPLF/XRPL-Standards/discussions) for the format of this memo

Reply:
do you mean the type "user-agent"? and the information getting passed in?

Reply:
Would the intent of that be to get more opinions on the inclusion of this? (And is there an example of what that might look like?)

Reply:
Yes, the type and its meaning.

We should probably follow existing standards for user agent identification. One big gap that I see here is that `xrpl-js` does not indicate the version of the library used. It would be ideal to incorporate the version number, e.g. `xrpl-js/2.8.0`.

Also, why is it `xrpl-js` instead of `xrpl.js`?

The intent would be to establish a standard that would be used by other client libraries and by other user agents. So, for example, a wallet like XUMM could set a user-agent of `XUMM/<version>` (or whatever format the XLS defines).

It would be fine to get some opinions, but that is not the primary intent.